orchestrator/src/main/java/com/scottlogic/datahelix/generator/orchestrator/generate/GenerateCommandLine.java

```diff
@@ -65,6 +66,12 @@ private void checkForAlphaGenerationDataTypes() {
         }
     }
 
+    @CommandLine.Option(
+        names = { "-V", "--version" },
+        versionHelp = true,
+        description = "Print version information and exit.")
+    boolean versionRequested;
+
```

Comment:
Adding this here will require the user to enter the command `datahelix generate -V` to get the version number. Is that what is intended?

I feel that perhaps a user would expect to be able to enter the command `datahelix -V` to get the version number.